src/v0/destinations/campaign_manager/transform.js

```diff
@@ -194,9 +201,110 @@ function process(event) {
   return response;
 }
 
+const generateBatch = (eventKind, events) => {
+  const batchRequestObject = defaultBatchRequestConfig();
+  const conversions = [];
+  let encryptionInfo = {};
+  const metadata = [];
+  // extracting destination, message from the first event in a batch
+  const { destination, message } = events[0];
+  // Batch event into dest batch structure
+  events.forEach((ev) => {
+    conversions.push(...ev.message.body.JSON.conversions);
```

Comment:
you could keep this ev.message.body.JSON in a variable and reuse it instead of typing long strings